wdio.conf.js

```diff
@@ -0,0 +1,78 @@
+var browserstack = require('browserstack-local')
+var browserlist = Object.values(require('./browserlist.json'))
+var args = process.argv.slice(2)
+var local = args.includes('--local')
+
+var build = require('child_process')
+  .execSync('git rev-parse --short=9 HEAD', { cwd: __dirname })
+  .toString().trim()
+
+var services = local ?
+  ['selenium-standalone', 'static-server'] :
+  ['browserstack','selenium-standalone','static-server']
+
+var capabilities = local ? [{
+  browserName: 'chrome'
+}] : browserlist.map(function (browser) {
+  return Object.assign({}, browser, {
+    'browserstack.local': true,
+    project: 'td-js-sdk',
+    build: build,
+  })
+})
+
+var localConfig = {
+  host: 'localhost',
+}
+
+var browserstackConfig = {
+  host: 'hub.browserstack.com',
+  user: process.env.BROWSER_STACK_USERNAME,
+  key: process.env.BROWSER_STACK_ACCESS_KEY,
+  // Code to start browserstack local before start of test
+  onPrepare: function (config, capabilities) {
```

Comment:
What's this local thing for?

Reply:
Tunnelling to browserstack so it can request localhost (the served fixtures). The karma plugin does this for us but the wdio one doesn't.